### PR TITLE
Optional Field Information in Task 1  |  Learning Exercise "Census"

### PR DESCRIPTION
**Pull request by SR8H**
The fact that age is an optional field is stated in task 2. This information should be specified in task 1 for clarity.

### CODE REVIEW

**Comment by junedev** [comment]
@SR8H Could you explain a bit more why you think the age being optional or not is relevant for the task?

**Review by junedev — COMMENTED** [review]
@SR8H Could you explain a bit more why you think the age being optional or not is relevant for the task?

**Comment by SR8H** [comment]
I think it's part of general information and not tied to task 1. Instructions for task 1 begin after this specifying that age is optional.

Specifying required and optional fields as early as possible is a better idea. Currently this information is in the middle of task 2, which I think could be clearer.

**Comment by junedev** [comment]
The general introduction is in line 1 to 6. The instructions for task 1 start with the headline for task 1 in line 7 and you added the new text below that. The problem I see is that now, students might think the information that the age is optional is relevant for solving task 1 which is not the case. They might try to add some check whether age is there, they might think the function is sometimes less arguments can be supplied in the constructor. It opens the doors for all kinds of confusion.

The tasks are designed so that only the information that is needed for the current section is given. From experience, that works a lot better then trying to cover all the information in the introduction part at the top as we did/do for some exercises in other tracks. This PR removes the information from the place where it is needed (task 2) and moves it to a place where it is not relevant.

In summary, I'm sorry but I don't think this PR improves the instructions. I will close it.

**Review by junedev — CHANGES_REQUESTED** [review]
The general introduction is in line 1 to 6. The instructions for task 1 start with the headline for task 1 in line 7 and you added the new text below that. The problem I see is that now, students might think the information that the age is optional is relevant for solving task 1 which is not the case. They might try to add some check whether age is there, they might think the function is sometimes less arguments can be supplied in the constructor. It opens the doors for all kinds of confusion.

The tasks are designed so that only the information that is needed for the current section is given. From experience, that works a lot better then trying to cover all the information in the introduction part at the top as we did/do for some exercises in other tracks. This PR removes the information from the place where it is needed (task 2) and moves it to a place where it is not relevant.

In summary, I'm sorry but I don't think this PR improves the instructions. I will close it.
